### PR TITLE
fix(UI): Fully handle default ack options

### PR DESCRIPTION
**Pull request by Jordan91**
## Description

This PR intends to fully handle default ACK options that can be set on Administration -> Parameters -> Monitoring

image

(ADDED: monitoring_default_acknowledgement_notify, monitoring_default_acknowledgement_with_services, monitoring_default_acknowledgement_force_active_checks)

**Fixes** # (issue)

## Type of change

 - [x] Patch fixing an issue (non-breaking change)

## Target serie

- [x] 21.04.x
- [x] 21.10.x
- [x] 22.04.x (master)

<h2> How this pull request can be tested ? </h2>

- Reach JIRA ticket for details

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

### CODE REVIEW

**Review by Thebarda — COMMENTED** [review]
Don't forget to complete the translations files

**Review by Thebarda — DISMISSED** [review]

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -16710,6 +16710,8 @@ msgstr "Se connecter avec"
 msgid "Can't resolve username from login claim using configured regular expression"
 msgstr "Impossible de résoudre le nom d'utilisateur à partir de la déclaration de connexion en utilisant l'expression régulière configurée"
 
+# msgid "Force active checks"
+# msgstr "Forcer les contrôles actif"
```
Comment (Thebarda) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:16714]:
```suggestion
msgid "Force active checks"
msgstr "Forcer les contrôles actifs"

```

File: lang/es_ES.UTF-8/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -15017,6 +15017,8 @@ msgstr ""
 # msgid "Login with"
 # msgstr ""
 
+# msgid "Force active checks"
+# msgstr ""
```
Comment (Thebarda) [inline lang/es_ES.UTF-8/LC_MESSAGES/messages.po:15021]:
```suggestion
# msgid "Force active checks"
# msgstr ""

```

File: lang/pt_BR.UTF-8/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -15468,6 +15468,8 @@ msgstr ""
 # msgid "Login with"
 # msgstr ""
 
+# msgid "Force active checks"
+# msgstr ""
```
Comment (Thebarda) [inline lang/pt_BR.UTF-8/LC_MESSAGES/messages.po:15472]:
```suggestion
# msgid "Force active checks"
# msgstr ""

```

File: lang/pt_PT.UTF-8/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -15456,6 +15456,8 @@ msgstr ""
 # msgid "Login with"
 # msgstr ""
 
+# msgid "Force active checks"
+# msgstr ""
```
Comment (Thebarda) [inline lang/pt_PT.UTF-8/LC_MESSAGES/messages.po:15460]:
```suggestion
# msgid "Force active checks"
# msgstr ""

```

File: www/front_src/src/Resources/Actions/Resource/Acknowledge/index.tsx [hunk 5]
```diff
@@ -59,10 +61,11 @@ const AcknowledgeForm = ({
 
   const form = useFormik<AcknowledgeFormValues>({
     initialValues: {
-      acknowledgeAttachedResources: false,
+      acknowledgeAttachedResources: acknowledgement.with_services,
       comment: undefined,
+      forceActiveCheck: acknowledgement.force_active_checks,
```
[the remark this answers is not in the document]
Comment (Jordan91) [reply www/front_src/src/Resources/Actions/Resource/Acknowledge/index.tsx]: Singular if you selected just one service or host

**Review by Thebarda — DISMISSED** [review]

**Review by jeremyjaouen — DISMISSED** [review]

**Review by kduret — DISMISSED** [review]

**Review by a-launois — DISMISSED** [review]

**Review by Thebarda — APPROVED** [review]

**Review by kduret — APPROVED** [review]

**Pull request merged**